### PR TITLE
To isis 

### PR DESCRIPTION
**Pull request by ladoramkershner**
Deleted redundant to_isis command and added capability to write out point's apriori and adjusted xyz to control network if they are constrained or fixed.

### CODE REVIEW

**Comment by ladoramkershner** [comment]
Gotta write some test, would be a good thing for me to do while autocnet chugs the elysium quad

File: autocnet/graph/network.py [hunk 1]
```diff
@@ -1472,6 +1470,30 @@ def to_isis(self, path, flistpath=None,sql = """
...
         df.rename(columns={'imageid':'image_index','id':'point_id', 'pointtype' : 'type',
             'sample':'x', 'line':'y', 'serial': 'serialnumber'}, inplace=True)
+
+        #create columns in the dataframe
+        df['aprioriX'] = 0
+        df['aprioriY'] = 0
+        df['aprioriZ'] = 0
+        df['adjustedX'] = 0
+        df['adjustedY'] = 0
+        df['adjustedZ'] = 0
```
Comment (jessemapel) [inline autocnet/graph/network.py]: You should comment that setting these to 0 means they won't get written out by protobuf in plio.

Comment (ladoramkershner) [reply autocnet/graph/network.py]: I added comments above the initialization, are these adequate?

Comment (jlaura) [reply autocnet/graph/network.py]: yes

**Pull request merged**